config.json

```diff
@@ -87,6 +87,9 @@
                 },
                 {
                     "name": "Migrating SDL Core 7.0 to 7.1"
+                },
+                {
+                    "name": "Migrating SDL Core 7.1 to 8.0"
```

Comment:
We might want to also add a link to this guide in the overview page https://smartdevicelink.com/en/guides/core/overview/#migrating-to-newer-sdl-versions